tests/Credentials/CredentialProviderTest.php

```diff
@@ -1076,6 +1077,87 @@ public function testSsoProfileProvider()
         }
     }
 
+    public function testSsoProfileProviderWithNewFileFormat()
+    {
+        $dir = $this->clearEnv();
+        $expiration = time() + 1000;
+        $ini = <<<EOT
+[profile default]
```

Comment:
Should be `[default]`

Reply:
I left this here to test that 'profile ' gets stripped and this gets used without having to add a basically duplicated test; should I get rid of it?